diff --git a/renderer.php b/renderer.php
index 6aba3f3..a899e62 100644
--- a/renderer.php
+++ b/renderer.php
@@ -16,6 +16,7 @@ require_once DOKU_INC.'inc/ZipLib.class.php';
  * The Renderer
  */
 class renderer_plugin_odt extends Doku_Renderer {
+    var $info = array("cache"=> false);
     var $ZIP = null;
     var $meta;
     var $store = '';
@@ -136,12 +137,9 @@ class renderer_plugin_odt extends Doku_Renderer {
 
         $this->_odtMeta();
 
-        // store the content type headers in metadata
-        $headers = array(
-            'Content-Type' => 'application/vnd.oasis.opendocument.text',
-            'Content-Disposition' => 'attachment; filename="'.noNS($ID).'.odt";'
-        );
-        p_set_metadata($ID,array('format' => array('odt' => $headers) ));
+        // send the content type header
+        header('Content-Type: application/vnd.oasis.opendocument.text');
+        header('Content-Disposition: attachment; filename="'.str_replace(':','-',$ID).'.odt";');
     }
 
     /**
